static/js/debugging.ts

```diff
@@ -86,11 +86,12 @@ export function load_variables(variables: any) {
     for (const i in variables) {
       // Only append if the variable contains any data (and is not undefined)
       if (variables[i][1]) {
-        variableList.append(`<li style=color:${variables[i][2]}>${variables[i][0]}: ${variables[i][1]}</li>`);
+        const variableName = variables[i][0].replace('_', '');
```

Comment:
I think we need to get rid of underscores if they appear in the beginning of the variable name. With the current approach, we remove the first underscore even when it is in the middle of the variable name, which is incorrect:
<img width="762" alt="Screenshot 2024-03-28 at 13 07 54" src="https://github.com/hedyorg/hedy/assets/3010723/1b5f3d43-176b-4e96-92d8-cff15f387440">